plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchConfig.java

```diff
@@ -79,15 +79,15 @@
     private Security security;
 
     @NotNull
-    public String getHost()
+    public String getHosts()
     {
-        return host;
+        return hosts;
     }
 
     @Config("elasticsearch.host")
-    public ElasticsearchConfig setHost(String host)
+    public ElasticsearchConfig setHosts(String hosts)
```

Comment:
This should be a `List<String>` not a String. The splitting on `,` will be done by the configuration system. It shouldn't be done by ElasticsearchClient. Otherwise, it's hard to do any validation and error reporting at startup time.

Reply:
Thanks for review! @martint  Did the changes, could you help review again? Thanks in advance!

Reply:
The above comment means changing the argument to `List<String>` instead of splitting by yourself.